lib/src/visitor/async_evaluate.dart

```diff
@@ -429,12 +435,24 @@ class _EvaluateVisitor
 
   /// Loads the module at [url] and passes it to [callback].
   ///
+  /// The [configuration] overrides values for `!default` variables defined in
+  /// the module or modules it forwards and/or imports. If it's not passed, the
+  /// current configuration is used instead. Throws a [SassRuntimeException] if
+  /// a configured variable is not declared with `!default`.
+  ///
   /// The [stackFrame] and [nodeForSpan] are used for the name and location of
   /// the stack frame in which the new module is executed.
   Future<void> _loadModule(Uri url, String stackFrame, AstNode nodeForSpan,
-      void callback(Module module)) async {
+      void callback(Module module),
+      {Map<String, _ConfiguredValue> configuration}) async {
     var builtInModule = _builtInModules[url];
     if (builtInModule != null) {
+      if (configuration != null || _configuration != null) {
```

Comment:
What exactly is this guarding against? Could the error message be more specific?

Reply:
This is guarding against someone passing in configuration to a built-in module, which never expose variables. I used the same error message as elsewhere because I want users to think of built-in modules as essentially the same kind of thing as user-defined modules, but if you think it would be substantially clearer I could change this to something like "Built-in modules can't be configured."

Reply:
Yeah I think that error with the updated message would be more clear and prevent someone from thinking they could possibly configure some built-in module if they just had the right variable in their config.

Reply:
Done.